webapp/src/pages/Map.tsx

```diff
@@ -654,108 +666,152 @@ function MapPage(props: MapPageProps) {
     return { value: type, label: type }
   })
 
+  const handleMenuSelection = (label: string) => {
+    if (menuSelection.includes(label)) {
+      setMenuSelection(menuSelection.filter((item) => item !== label))
+      switch (label) {
+        case 'Display Message Counts':
+          dispatch(setDisplay({ view: 'tab', display: '' }))
+          break
+        case 'Display RSU Status':
+          dispatch(setDisplay({ view: 'tab', display: '' }))
+          break
+      }
+    } else {
+      setMenuSelection([...menuSelection, label])
```

Comment:
The "Display Message Counts" and "Display RSU Status" options should override each other. The UI allows both to be selected, when only the most recently toggled one is active. They should toggle each other to make it more obvious which one is active